apps/explorer/src/pages/transaction-result/programmable-transaction-view/Inputs.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiJsonValue, toB64 } from '@mysten/sui.js';
+import clsx from 'clsx';
+
+import styles from '~/pages/transaction-result/TransactionResult.module.css';
+import { AddressOrObject } from '~/pages/transaction-result/programmable-transaction-view/AddressOrObject';
+import { TableHeader } from '~/ui/TableHeader';
+
+interface Props {
+    inputs: SuiJsonValue[];
+}
+
+export function Inputs({ inputs }: Props) {
+    if (!inputs?.length) {
+        return null;
+    }
+
+    return (
+        <div data-testid="programmable-transactions-inputs" className="mt-8">
+            <section
+                className={clsx([styles.txcomponent, styles.txgridcolspan2])}
+            >
+                <TableHeader>Inputs</TableHeader>
+                <ul className="flex flex-col gap-y-3">
+                    {inputs.map((input) => {
+                        if (Array.isArray(input)) {
+                            const readableInput = toB64(
+                                input as unknown as Uint8Array
```

Comment:
I'd probably actually convert this to a uint8array instead of typecast

Reply:
Problem is the `SuiJsonValue` type is a union of a bunch of types. Doing `Array.isArray` does not coerce the types on subsequent lines to be `ArrayBufferLike`. Even if I update the actual types in `common.ts` for `SuiJsonValue`, it still does not work correctly.
Open for suggestion here

<img width="1184" alt="Screenshot 2023-03-15 at 10 39 32 PM" src="https://user-images.githubusercontent.com/127577476/225525142-163f8fb7-f503-48b3-a9ea-cdb7bc376fb4.png">